package.json

```diff
@@ -81,6 +81,9 @@
           "<rootDir>/lib/.*",
           "<rootDir>/src/rules/__tests__/fixtures/*",
           "<rootDir>/src/rules/__tests__/test-utils.ts"
+        ],
+        "coveragePathIgnorePatterns": [
+          "<rootDir>/tools/"
```

Comment:
what isn't covered?

Reply:
It's the tools - becuase they're not imported by anything that is involved in the tests, they're not expected to be covered. When @bmish introduced tests that imported from there, they got included in coverage.

Now that those tests have been axed again, this can go too

Reply:
Removed.